controllers/lvmcluster_controller.go

```diff
@@ -183,7 +186,8 @@ func (r *LVMClusterReconciler) reconcile(ctx context.Context, instance *lvmv1alp
 	for _, unit := range resourceCreationList {
 		err := unit.ensureCreated(r, ctx, instance)
 		if err != nil {
-			return ctrl.Result{}, fmt.Errorf("failed reconciling: %s %w", unit.getName(), err)
+			r.Log.Error(err, "failed reconciling", "Resource", unit.getName())
```

Comment:
This is actually changing the error being returned, right?

Reply:
yes I know, We should return an original error IMO as that can be used to determine the error in the caller's place.